apps/crn-frontend/src/shared-research/__tests__/export.test.ts

```diff
@@ -66,6 +66,7 @@ describe('researchOutputToCSV', () => {
       organisms: 'C. Elegans,Rat',
       environments: 'In Cellulo,In Vivo',
       subtype: 'Metabolite',
+      publishingEntity: 'Team',
```

Comment:
Not sure if I follow it but wouldn't publishingEntity be 'Shared Research' instead of Team?

Reply:
No, a shared research can have one of two publishing entities, either Team or Working Group